src/components/bluepad32/parser/uni_hid_parser_ds5.c

```diff
@@ -195,6 +200,94 @@ static void ds5_request_calibration_report(uni_hid_device_t* d);
 static void ds5_set_rumble_off(btstack_timer_source_t* ts);
 static void ds5_parse_mouse(uni_hid_device_t* d, const uint8_t* report, uint16_t len);
 
+
+/******* Dualsense(DS5) Adaptive Trigger Effects - Start *******/
+// Built with help of https://gist.github.com/Nielk1/6d54cc2c00d2201ccb8c2720ad7538db, licensed under MIT License
+
+// Switches trigger effect off
+void ds5_generate_trigger_effect_off(uint8_t effect[11]) {
+    effect[0] = 0x05;
+
+    for (uint8_t i = 1; i < 8; i++)
```

Comment:
should be i < 11, typo

Reply:
Thanks. Could you update the pull-request so that it uses `11` instead of `8` ?